.travis.yml

```diff
@@ -27,6 +27,8 @@ virtualenv:
   system_site_packages: true
 
 install:
+  - git clone https://github.com/OCA/reporting-engine ${HOME}/reporting-engine -b ${VERSION}
+  - git clone https://github.com/OCA/web ${HOME}/web -b ${VERSION}
```

Comment:
You can put your dependencies in a file named `oca_dependencies.txt` (sample https://github.com/acsone/l10n-belgium/blob/8.0-transifex-sbi/oca_dependencies.txt)